contract/tsconfig.json

```diff
@@ -1,16 +1,19 @@
 {
   "compilerOptions": {
     "target": "ESNext",
-    "module": "ESNext",
-    "moduleResolution": "node",
+    "module": "NodeNext",
+    "moduleResolution": "NodeNext",
     "lib": ["ESNext", "DOM"],
-    "strict": true,
+    "checkJs": true,
+    "strict": false,
+    "noImplicitAny": false,
+    "useUnknownInCatchVariables": false,
```

Comment:
this looks like some hard-won knowledge.

Something to add to https://github.com/Agoric/agoric-sdk/discussions/8274 ?

Reply:
they manifest pretty loudly. `useUnknownInCatchVariables` and `noImplicitAny` were relaxing `strict: true` but there were so many other hurdles I just turned that off (it's off in sdk). I'll remove these two redundant flags

Reply:
oh, most of the other problems are solved so I'll flip it to true and suppress a few